bot/exts/evergreen/latex.py

```diff
@@ -0,0 +1,77 @@
+import re
```

Comment:
Nitpicking but I think you'll want to rearrange those imports according to alphabetical order as well

Reply:
the imports are pretty much in the order for which flake8 doesn't complain, probably because `import...`s should go before `from ... import ...`s